render-test/runner.cpp

```diff
@@ -629,7 +629,7 @@ TestOperations getAfterOperations(const Manifest& manifest) {
         }
         if (networkProbeOp == probe) {
             result.emplace_back([](TestContext& ctx) {
-                assert(ProxyFileSource::isTrackingActive());
+//                assert(ProxyFileSource::isTrackingActive());
```

Comment:
```suggestion
```

Reply:
Thanks for review，the comment is deleted